docs_user/modules/proc_migrating-existing-daemons-to-target-nodes.adoc

```diff
@@ -140,15 +131,11 @@ mgr  advanced  mgr/dashboard/ALERTMANAGER_API_HOST   http://172.17.3.83:9093
 mgr  advanced  mgr/dashboard/PROMETHEUS_API_HOST     http://172.17.3.83:9092
 mgr  advanced  mgr/dashboard/GRAFANA_API_URL         https://172.17.3.144:3100
 ----
-
-. The Ceph Dashboard (mgr module plugin) has not been impacted at all by this
-relocation. The service is provided by the Ceph Manager daemon, hence we might
-experience an impact when the active mgr is migrated or is force-failed.
-However, having three replicas definition allows to redirect requests to a
-different instance (it’s still an A/P model), hence the impact should be
++
+[NOTE]
+The Ceph Dashboard (mgr module plugin) is not impacted by the relocation. The service is provided by the Ceph Manager daemon. You might experience an impact when the active `mgr` daemon is migrated or is force-failed. However, defining 3 replicas allows you to redirect requests to a different instance (it’s still an A/P model), hence the impact should be
 limited.
-.. When the RBD migration is over, the following {Ceph} config keys must
-be regenerated to point to the right mgr container:
+. When the RBD migration is over, regenerate the following {Ceph} configuration keys to point to the right `mgr` container:
```

Comment:
We mention RBD migration, but that is done in a different chapter that I believe comes after this one.

Reply:
correct, while here we migrate the monitoring stack, the ceph dashboard (intended as the Ceph Manager daemon plugin), is implicitly migrated to target nodes together with the ceph-mgr, covered in the rbd section.

Reply:
@fmount Does that mean that customers should complete this specific step after they complete the RBD migration procedure? 
Or should RBD migration be a prerequisite to migrating the monitoring stack?

Reply:
No, the Ceph dashboard (not the monitoring stack) will be implicitly migrated when we do RBD, but that part is neither a requirement nor something that should be done before the monitoring stack.
For any configuration that needs to be updated, we can provide a `post` procedure after the final verification of the status of the cluster.
Do you think we should move this particular point to that `post` section?

Reply:
Yes, I think we should create a "post" procedure. It's a little confusing that this step appears before the customer does the RBD migration. So they would run through this procedure, then do RBD, then have to come back to this procedure. 

I'll hide it and we'll follow up in another PR, as you suggested.

Reply:
@fmount Do you want to create a Jira ticket for this so we don't forget, or do you want me to create it?

Reply:
If you want to create one feel free, good to keep track of the remaining work, otherwise I can create one tomorrow my morning

Reply:
I'll create one.

Reply:
Thank you!